.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Debug MaxText Decode",
+      "type": "python",
+      "request": "launch",
+      "console": "integratedTerminal",
+      "justMyCode": false,
+      "python": "python3",
+      "program": "${workspaceFolder}/MaxText/decode.py",
+      "args": ["MaxText/configs/base.yml", 
+               "run_name=runner_$(date +%Y-%m-%d-%H-%M)", 
+               "base_output_directory=gs://patemotter-maxtext-out",
```

Comment:
nit: can you make this one of the shared directories? and while you're being a hero can you add a a train version?

Reply:
Done